flexmeasures/ui/templates/crud/asset.html

```diff
@@ -34,6 +34,12 @@
     </div>
     <div class="row">
         <div class="col-sm-2 on-top-md">
+            <div>
```

Comment:
I think this should not have its own line and not be as high.

Can you place it next to "Asset status"? (And that can be shortened to say "Status")

Reply:
Done, but status button is still "Asset status" as when it is "Status" this results in status button being thinner than audit log button